Jamulus.pro

```diff
@@ -1,5 +1,10 @@
 VERSION = 3.10.0dev
 
+# Using lrelease and embed_translations only works for Qt 5.12 or later
```

Comment:
Please link this PR here. This would users allow to see that they could in theory build using an older Qt version if they revert this.

Reply:
OK, done. Will be in my next push.